datafusion/sql/tests/sql_integration.rs

```diff
@@ -43,8 +43,8 @@ fn parse_decimals() {
     let test_data = [
         ("1", "Int64(1)"),
         ("001", "Int64(1)"),
-        ("0.1", "Decimal128(Some(1),1,1)"),
-        ("0.01", "Decimal128(Some(1),2,2)"),
+        ("0.1", "Decimal128(Some(1),2,1)"),
+        ("0.01", "Decimal128(Some(1),3,2)"),
```

Comment:
Previous tests were wrong.

```bash
D select cast(.1 as decimal(2,1));
┌──────────────────────────┐
│ CAST(.1 AS DECIMAL(2,1)) │
├──────────────────────────┤
│ 0.1                      │
└──────────────────────────┘
D select cast(.1 as decimal(1,1));
┌──────────────────────────┐
│ CAST(.1 AS DECIMAL(1,1)) │
├──────────────────────────┤
│ .1                       │
└──────────────────────────┘
```

Reply:
🤔 I don't understand why they were wrong

I think it is possible to fit `0.1` into a Decimal(1,1):

```
❯ select arrow_cast(0.1, 'Decimal128(1,1)') as d;
+-----+
| d   |
+-----+
| 0.1 |
+-----+
```

Reply:
Sorry, it is my fault. I checked it by Duckdb.